ajv.js

```diff
@@ -7,29 +7,22 @@ const ajvFormats = require('ajv-formats')
 module.exports = buildAjv
 
 function buildAjv (options) {
-  const ajvInstance = new Ajv({ ...options, strictSchema: false, validateSchema: false, uriResolver: fastUri })
-  ajvFormats(ajvInstance)
+  const ajvInstance = new Ajv({
+    ...options,
+    strictSchema: false,
+    validateSchema: false,
+    allowUnionTypes: true,
+    uriResolver: fastUri
+  })
 
-  const validateDateTimeFormat = ajvFormats.get('date-time').validate
-  const validateDateFormat = ajvFormats.get('date').validate
-  const validateTimeFormat = ajvFormats.get('time').validate
+  ajvFormats(ajvInstance)
 
   ajvInstance.addKeyword({
-    keyword: 'fjs_date_type',
-    validate: (schema, date) => {
-      if (date instanceof Date) {
-        return true
-      }
-      if (schema === 'date-time') {
-        return validateDateTimeFormat(date)
-      }
-      if (schema === 'date') {
-        return validateDateFormat(date)
-      }
-      if (schema === 'time') {
-        return validateTimeFormat(date)
-      }
-      return false
+    keyword: 'fjs_type',
```

Comment:
```suggestion
    keyword: 'fjs_date_type',
```


I think we should try not to change this.

Reply:
@mcollina I agree, but it is a completely different meaning of this property now. It's not a date format anymore. It describes how ajv should validate custom type at all. For example, for string type, there would be not only the Date object validation but also Regexp objects and some other exceptions.